test/Microsoft.Dnx.TestHost.Tests/ProjectTest.cs

```diff
@@ -20,13 +20,11 @@ public class ProjectTest
 
         public ProjectTest()
         {
-            var libraryManager = PlatformServices.Default.LibraryManager;
-
-            // Example: "C:\github\testing\test\Sample.Tests"
-            _testProjectPath = Path.GetDirectoryName(libraryManager.GetLibrary(projectName).Path);
+            var testProjectFolder = Path.GetDirectoryName(Path.GetDirectoryName(Path.GetDirectoryName(Directory.GetCurrentDirectory())));
+            _testProjectPath = Path.Combine(testProjectFolder, projectName);
         }
 
-        [Fact]
+        [Fact(Skip = "Doesn't run on dotnet")]
```

Comment:
This doesn't tell us anything. _Why_ doesn't this run on `dotnet`? Could the test have conditions that allow it to run on at least a few system / platform combinations/